aten/src/ATen/native/mps/operations/RMSNorm.mm

```diff
@@ -0,0 +1,67 @@
+#define TORCH_ASSERT_ONLY_METHOD_OPERATORS
+
+#ifndef AT_PER_OPERATOR_HEADERS
+#include <ATen/Functions.h>
+#include <ATen/NativeFunctions.h>
+#else
+#include <ATen/ops/empty_like.h>
+#endif
+#include <ATen/mps/MPSProfiler.h>
+#include <ATen/native/mps/OperationUtils.h>
+#include <ATen/native/mps/operations/RMSNorm.h>
+#include <fmt/format.h>
+
+namespace at::native::mps {
+
+#ifndef PYTORCH_JIT_COMPILE_SHADERS
+static auto& lib = MetalShaderLibrary::getBundledLibrary();
+#else
+#include <ATen/native/mps/RMSNorm_metallib.h>
+#endif
+
+Tensor rms_norm_mps_kernel(const Tensor& input,
+                           c10::SymIntArrayRef normalized_shape,
+                           const Tensor& weight,
+                           const double eps) {
+  auto output = at::empty_like(input);
```

Comment:
Nit (i.e. if function only works with contigous inputs no harm in checking for those
```suggestion
  TORCH_CHECK(input.is_contigous() && weight.is_contigous());
  auto output = at::empty_like(input);
```